.drone.yml

```diff
@@ -109,17 +109,17 @@ pipeline:
 
 matrix:
   include:
-    - { PHP_VERSION: 8.0, CORE_VERSION: 9.4.0, TYPE: phpunit }
-    - { PHP_VERSION: 8.0, CORE_VERSION: 9.4.0, TYPE: behat, BATCH: '@batch1' }
-    - { PHP_VERSION: 8.0, CORE_VERSION: 9.4.0, TYPE: behat, BATCH: '@batch2' }
-    - { PHP_VERSION: 8.0, CORE_VERSION: 9.4.0, TYPE: behat, BATCH: '@batch3' }
-
-    - { PHP_VERSION: 8.1, CORE_VERSION: 9.4.0, TYPE: phpunit }
-    - { PHP_VERSION: 8.1, CORE_VERSION: 9.4.0, TYPE: behat, BATCH: '@batch1' }
-    - { PHP_VERSION: 8.1, CORE_VERSION: 9.4.0, TYPE: behat, BATCH: '@batch2' }
-    - { PHP_VERSION: 8.1, CORE_VERSION: 9.4.0, TYPE: behat, BATCH: '@batch3' }
-
     - { PHP_VERSION: 8.1, CORE_VERSION: 10.0, TYPE: phpunit }
     - { PHP_VERSION: 8.1, CORE_VERSION: 10.0, TYPE: behat, BATCH: '@batch1' }
     - { PHP_VERSION: 8.1, CORE_VERSION: 10.0, TYPE: behat, BATCH: '@batch2' }
     - { PHP_VERSION: 8.1, CORE_VERSION: 10.0, TYPE: behat, BATCH: '@batch3' }
+
+    - { PHP_VERSION: 8.2, CORE_VERSION: 10.0, TYPE: phpunit }
+    - { PHP_VERSION: 8.2, CORE_VERSION: 10.0, TYPE: behat, BATCH: '@batch1' }
+    - { PHP_VERSION: 8.2, CORE_VERSION: 10.0, TYPE: behat, BATCH: '@batch2' }
+    - { PHP_VERSION: 8.2, CORE_VERSION: 10.0, TYPE: behat, BATCH: '@batch3' }
+
+    - { PHP_VERSION: 8.2, CORE_VERSION: 10.1, TYPE: phpunit }
```

Comment:
we should add also 8.1 and 10.1. Before the matrix had only 3 entries as 10.x was not compatible with 8.0.